src/app/runbox-components/runbox-section.ts

```diff
@@ -47,39 +47,33 @@ export class RunboxSectionComponent {
     h1: {
     },
     h2: {
-        margin: '0 10px 50px 10px',
+        margin: '0 10px 50px 0px',
     },
     h3: {
-        margin: '0 10px 50px 10px',
+        margin: '0 10px 50px 0px',
     },
     h4: {
-        margin: '0 10px 50px 10px',
+        margin: '0 10px 50px 0px',
     },
     h5: {
-        margin: '0 10px 50px 10px',
+        margin: '0 10px 50px 0px',
     },
     h6: {
-        margin: '0 10px 50px 10px',
+        margin: '0 10px 50px 0px',
     },
   };
   public heading_sizes = {
     h1: {
-        background: '#CAE1FF'
     },
     h2: {
-        background: '#e1eeff'
     },
     h3: {
-        background: '#9CC6FE'
     },
     h4: {
-        background: '#7BB1FA'
     },
     h5: {
-        background: '#549CFC'
     },
     h6: {
-        background: '#207efe'
     },
   };
```

Comment:
After removing all those background colors, the whole `heading_sizes` block isn't needed there anymore. Maybe it's best to just remove the whole thing, keeping it clean :)

Reply:
That's true, but it might be useful with the placeholders if we'd like to add styles in the future. :)